static_test_driver/static_test_driver.ino

```diff
@@ -18,14 +18,19 @@ float chamber_temp, inlet_temp, outlet_temp,pressure,x,y,z;
 bool temp_status = false;
 #define CONFIGURATION DEMO
 
-//Thermocouple Setup
+
 #if CONFIGURATION == MK_2
 #define MAXDO1   3
 #define MAXCS1   4
 #define MAXCLK1  5
 Adafruit_MAX31855 Chamber_Thermocouple(MAXCLK1, MAXCS1, MAXDO1);
+//Pressure Setup
+#define PRESSURE_CALIBRATION_FACTOR 246.58
+#define PRESSURE_OFFSET 118.33
+#define PRESSURE_PIN 1
 #endif
 
+//Thermocouple Setup
```

Comment:
I thought I would put all the code for the sensors used solely in MK_2 together under one #if  even if it means separating sensors of the same kind.